class.FlipPage.php

```diff
@@ -397,11 +401,11 @@ protected function addAllLinks()
             $this->addLink('Logout', $this->logoutUrl);
         }
         $aboutMenu = array(
```

Comment:
Is the about menu really a priority you need to change? I'd really rather do it right than piecemeal...